lib/rubocop/cop/style/rescue_modifier.rb

```diff
@@ -67,9 +67,15 @@ def parenthesized?(node)
           node.parent && parentheses?(node.parent)
         end
 
+        # rubocop:disable Metrics/AbcSize
         def correct_rescue_block(corrector, node, parenthesized)
           operation = node.body
 
+          if operation.array_type? && !operation.bracketed?
+            corrector.insert_before(operation, '[')
+            corrector.insert_after(operation, ']')
+          end
```

Comment:
```suggestion
          corrector.wrap(operation, '[', ']') if operation.array_type? && !operation.bracketed?
```

Reply:
Thanks, I was wanting this but forgot about it